docs/scripts/generate_skills_docs.py

```
@@ -39,18 +51,120 @@ class Skill:
+
+    yml_content = _fetch_internal_file("fiftyone.yml", token)
+    if not yml_content:
+        logger.warning(
+            "Could not fetch fiftyone.yml from fiftyone-internal-skills"
+        )
+        return []
+
+    try:
+        plugin_config = yaml.safe_load(yml_content) or {}
+    except yaml.YAMLError as e:
+        logger.warning(f"Failed to parse fiftyone.yml: {e}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)